common/decklist.cpp

```diff
@@ -562,9 +562,9 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         }
 
         // Filter out MWS edition symbols and basic land extras
-        QRegExp rx("\\[.*\\]");
+        QRegExp rx("\\[.*\\]\s?");
         line.remove(rx);
-        rx.setPattern("\\(.*\\)");
+        rx.setPattern("\\\s(.*\\)");
```

Comment:
Needs the `\` within `\s` another `escape \` infront?
--> `\\s`?

Reply:
Actually, @dino572 fixed this and managed to created his own PR: https://github.com/Cockatrice/Cockatrice/pull/3057